test/Base.t.sol

```diff
@@ -51,19 +54,28 @@ abstract contract Base_Test is Assertions, Calculations, DeployOptimized, Modifi
 
     function setUp() public virtual {
         // Deploy the base test contracts.
+        contractWithoutReceive = new ContractWithoutReceive();
+        contractWithReceive = new ContractWithReceive();
         dai = new ERC20Mock("Dai Stablecoin", "DAI");
         noop = new Noop();
         recipientGood = new RecipientGood();
         usdt = new ERC20MissingReturn("Tether USD", "USDT", 6);
 
+        // Fund the contracts that need it.
+        vm.deal({ account: address(contractWithoutReceive), newBalance: 100 ether });
```

Comment:
Since they receive eth, they do not need to be funded with eth

Reply:
https://github.com/sablier-labs/v2-core/pull/1093/commits/13d8af9906bb19e767fc608543703d004856559f

Reply:
I used to make those contracts the `msg.sender` in the `wtihdrawFees` tests, but I then switched to Alice. In any case, it works now without funding them with ETH.